picard/mbjson.py

```diff
@@ -366,7 +366,8 @@ def recording_to_metadata(node, m, track=None):
             # set tags from artists
             if track:
                 for artist in value:
-                    track.append_track_artist(artist['artist']['id'])
+                    artist_obj = track.append_track_artist(artist['artist']['id'])
+                    add_genres_from_node(artist['artist'], artist_obj)
```

Comment:
I think it'd be more readable to use another intermediate variable, something like:

```python
                for artist in value:
                    art = artist['artist']
                    artist_obj = track.append_track_artist(art['id'])
                    add_genres_from_node(art, artist_obj)
```

Reply:
Done, also added unit test